.pre-commit-config.yaml

```diff
@@ -57,6 +57,13 @@ repos:
       language: pygrep
       types: [python]
       exclude: ^src/sentry/metrics/minimetrics.py$
+    - id: prevent-push
+      name: prevent pushing master
+      stages: [push]
```

Comment:
this should be `pre-push` now in newer pre-commit -- and add https://pre-commit.com/#top_level-default_install_hook_types to the top level so it gets installed as well

Reply:
I think I've done this - one thing I'm unsure of is whether `default_install_hook_types` needs "pre-commit" too.